osmaxx-py/osmaxx/excerptexport/static/excerptexport/stylesheets/main.css

```diff
@@ -223,8 +223,10 @@ table tr:nth-child(odd) td  { background: rgb(210,240,255); }
 [data-order-status="PROCESSING"] { background: lightcyan; border: 0.1rem solid turquoise; }
 [data-order-status="FINISHED"] { background: honeydew; border: 0.1rem solid seagreen; }
 [data-order-status="CANCELED"] { background: peachpuff; border: 0.1rem solid maroon; }
+[data-order-status="FAILED"] { background: mistyrose; border: 0.1rem solid maroon; }
```

Comment:
should it be the same color for canceled and failed, with just a change in background (I guess I'm asking, if this is on purpose).